docs/devel/automation.md

```diff
@@ -80,7 +80,7 @@ A PR is considered "ready for merging" if it matches the following:
      * Jenkins GCE e2e
      * Jenkins unit/integration
   * The PR cannot have any prohibited future milestones (such as a v1.5 milestone during v1.4 code freeze)
-  * The PR must have the "lgtm" label
+  * The PR must have a review comment "LGTM" (case-insensitive).
```

Comment:
nit: I think "The PR must have the "lgtm" label.  The "lgtm" label is automatically applied ..."  is slightly clearer than the compound sentence.

And perhaps "comment consisting of only "LGTM" (case-insensitive)" or some such, since as the code reads, there can't be anything in the comment other than LGTM.